src/calculator.js

```diff
@@ -1,48 +1,31 @@
+module.exports = exports;
+var x;
+var y;
```

Comment:
Unexpected var, use let or const instead.

###### See http://eslint.org/docs/rules/no-var for details about this rule